### PR TITLE
Feature/offline reading of lists

### PR DESCRIPTION
**Pull request by aseehra**
This requires another `npm install` because I use lodash/throttle.

This is a bit more hacky than I would have liked in places, but it will do for now.

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import LogOut from './auth-components/logout';
+import ConnectivityMessage from './ConnectivityMessage';
 import './component.css';
+
 export class NavBar extends React.Component {
   render() {
     let logOutButton;
```
Comment (ender554) [inline src/components/NavBar.js]: May want to disable logout button when not online

Comment (ender554) [inline src/components/NavBar.js]: Also add button seems to be gone now or rather the plus image, not sure if from this branch or a previous this was merged from. 

Also delete list button not working, but may be from previous branch.

Comment (aseehra) [reply src/components/NavBar.js]: The add button being gone is unrelated to this PR (check dev/master). Can you explain delete list not working more?

Comment (aseehra) [reply src/components/NavBar.js]: Nevermind, I can reproduce. Added to my todo list

**Comment by aseehra** [comment]
Updated to reflect @ender554's comments.

**Pull request closed without merging**